src/graphql/v1/generator/search_parameters.py

```diff
@@ -6430,7 +6430,7 @@
 		'name': {
 			'description': 'A server defined search that may match any of the string fields in the HumanName, including family, give, prefix, suffix, suffix, and/or text',
 			'type': 'string',
-			'field': 'name',
+			'fields': ['name.text', 'name.given', 'name.family', 'name.suffix', 'name.prefix'],
```

Comment:
This field array looks to be used throughout the changes, maybe we could make this into a variable and reuse it?